src/Compilers/Core/Portable/Symbols/CompilerFeatureRequiredHelpers.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection.Metadata;
+
+namespace Microsoft.CodeAnalysis;
+
+[Flags]
+internal enum CompilerFeatureRequiredFeatures
+{
+    None = 0,
+    RefStructs = 1 << 0,
+    RequiredMembers = 1 << 1,
+
+    Unknown = 1 << 31
```

Comment:
>Unknown = 1 << 31 [](http://example.com/codeflow?start=4&length=17)

What is the benefit in having this member? I think the assert below isn't really that useful and returning ```None``` from ```getFeatureKind``` would work as good.

Reply:
I think the logic is clearer not using `None` for this.